### DISCUSSION TITLE
Qual o código para reproduzir esse site? HTML e CSS

### DISCUSSION THREAD

**Discussion by mememememe7** [frontendbr/forum#2483] [Perguntas e Respostas]
Qual o código para reproduzir esse site???
Com adaptação WEB
![full](https://github.com/frontendbr/forum/assets/168691230/c0f0d3e1-0403-42cd-b266-53bcf7a0659d)

**Comment by nazarepiedady** [comment]
@mememememe7, use a ferramenta de programação do navegador chamada **inspetor de elementos** para inspecionares a marcação de hipertexto (HTML) e as folhas de estilos em cascata (CSS) usadas pelo programador do sítio da Web em que estás interessado em replicar algum componente que nele existe.
